src/modules/dashboard/components/Wallet/UserTokenEditDialogForm.tsx

```diff
@@ -0,0 +1,222 @@
+import React, { useCallback, useState, useMemo } from 'react';
+import { FormikProps, FormikHelpers } from 'formik';
+import { defineMessages, FormattedMessage, useIntl } from 'react-intl';
+import * as yup from 'yup';
+import { AddressZero } from 'ethers/constants';
+import isEqual from 'lodash/isEqual';
+
+import Button from '~core/Button';
+import Dialog, { DialogSection } from '~core/Dialog';
+import { Form } from '~core/Fields';
+import Heading from '~core/Heading';
+import Paragraph from '~core/Paragraph';
+import TokenSelector from '~dashboard/CreateColonyWizard/TokenSelector';
+import TokenItem from '~core/TokenEditDialog/TokenItem/index';
+
+import { UserTokens, OneToken, useLoggedInUser } from '~data/index';
+import { Address } from '~types/index';
+import { createAddress } from '~utils/web3';
+
+import styles from './UserTokenEditDialogForm.css';
+
+const MSG = defineMessages({
+  title: {
+    id: 'dashboard.UserTokenEditDialog.UserTokenEditDialogForm.title',
```

Comment:
I think those ids are to change, UserTokenEditDialog is not needed here

Reply:
Why? I thought that this was a convention that we are using, e.g.: 
`dashboard.TransferFundsDialog.TransferFundsDialogForm.title` (and also other dialog forms)
I am happy to change, just want to understand what would be the difference in this case.

Reply:
Alicja is right here. this should be:
```
dashboard.Wallet.UserTokenEditDialogForm.title
```

Also the component should have a `displayName`, which should be `dashboard.Wallet.UserTokenEditDialogForm`